src/openexr.imageio/exrinput.cpp

```diff
@@ -1024,6 +1024,9 @@ OpenEXRInput::seek_subimage(int subimage, int miplevel)
     m_miplevel = miplevel;
     m_spec     = part.spec;
 
+    if (!check_open(m_spec, { 0, 1 << 30, 0, 1 << 30, 0, 1 << 16, 0, 1 << 16 }))
```

Comment:
`1 << 30` seems high to me. Maybe `1 << 20`? I think we should run with the assumption that we're not going to encounter an OpenEXR file that is more than 1 million pixels wide or high, and that if a file says it is, it's most likely corrupted somehow. Similarly `1 << 16` is an awful lot of channels. For OpenEXR output, I used `1 << 12` which is still pretty generous.